local-setup/jupyter/dataloader/crs_loader.py

```
@@ -1378,6 +1378,14 @@ def load_hierarchy(self, name: str, levels: list, target_tenant: str = None,
             print(f"   ERROR: Template generation failed")
             return None
 
+        print(f"\n[3/5] Loading boundary level localizations...")
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)